client/X11/xf_cliprdr.c

```diff
@@ -31,13 +31,15 @@
 #include <X11/extensions/Xfixes.h>
 #endif
 
-#ifdef WITH_FUSE
-#define FUSE_USE_VERSION FUSE_API_VERSION
-#if FUSE_USE_VERSION >= 30
-#include <fuse3/fuse_lowlevel.h>
-#else
-#include <fuse/fuse_lowlevel.h>
+#ifdef WITH_FUSE3
+#define FUSE_USE_VERSION 30
```

Comment:
We can directly go with version `35` here.

Reply:
no need for that, we just need the minimum version we support here.